ptx/sec_deriv_inverse_function.ptx

```diff
@@ -378,7 +399,12 @@
   <sidebyside widths="47% 47%" margins="0%">
       <!-- START figures/fig_inverse4.tex -->
       <image xml:id="img_inverse4">
-        <description></description>
+      <shortdescription>graph of sin x and sin^{-1} x with corresponding tangent lines</shortdescription>
+        <description>
+        <p> starting at <m>(-pi/2,0)</m> the graph starts moving to the right for a short distance then starts sloping upwards to the right. It continues upwards through the
+        origin until it reaches t the local maximum. The tangent line of the curve is drawn at <m>(pi/3,sqrt(3)/2)</m>.
```

Comment:
I think this long description needs a bit of work. It doesn't seem to agree with the short description?
"reaches t" is a typo, I think. And you will want to avoid "local maximum" since this is not defined until the next chapter.